src/app/components/core/notifications/add-notification-wizard/add-notification-wizard.component.ts

```diff
@@ -351,7 +351,7 @@ export class AddNotificationWizardComponent implements OnInit, OnDestroy {
     this.isRulePlugin = true;
     this.isDeliveryPlugin = true;
     const plugin = (selectedPlugin.slice(3).trim()).replace(/'/g, '');
-    if (pluginType === 'notificationRule') {
+    if (pluginType === 'rule') {
```

Comment:
Having this conditional block with `pluginType === 'rule' || pluginType === 'notificationRule'`  OR `['rule', 'notificationRule'].includes(pluginType)` is also okay.